tests/FixerDefinition/FileSpecificCodeSampleTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\FixerDefinition;
+
+use PhpCsFixer\FixerDefinition\FileSpecificCodeSample;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @author Andreas Möller <am@localheinz.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\FixerDefinition\FileSpecificCodeSample
+ */
+final class FileSpecificCodeSampleTest extends TestCase
+{
+    public function testImplementsFileSpecificCodeSampleInterface()
+    {
+        $sample = new FileSpecificCodeSample(
+            file_get_contents(__FILE__),
+            new \SplFileInfo(__FILE__)
+        );
+
+        $this->assertInstanceOf('PhpCsFixer\FixerDefinition\FileSpecificCodeSampleInterface', $sample);
+    }
+
+    public function testDefaults()
+    {
+        $sample = new FileSpecificCodeSample(
+            file_get_contents(__FILE__),
+            new \SplFileInfo(__FILE__)
+        );
+
+        $this->assertNull($sample->getConfiguration());
```

Comment:
Doing this below, this is just to demonstrate that `$configuration` is an optional parameter and that its value defaults to `null` unless provided.